src/sonic-config-engine/minigraph.py

```diff
@@ -1530,6 +1530,12 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         if inband_port in ports.keys():
             ports[inband_port]['admin_status'] = 'up'
 
+# bring up the recirc port for voq chassis
+    for port, port_attributes in ports.items():
+        port_role = port_attributes.get('role', None)
+        if port_role == 'Rec':
+            ports[port]['admin_status'] = 'up'
```

Comment:
>ports[port]['admin_status'] [](http://example.com/codeflow?start=12&length=27)

If the original value is `down`, do you intend to override?

Reply:
This is an internal port. There is no configuration for this port in minigraph, currently. We want to set this port to `admin-status` as `up`  always for everflow to work on Voq Chassis